dogowner/test_dog_owner.py

```diff
@@ -0,0 +1,58 @@
+from .models import Dogowner
+from django.contrib.auth.models import User
+import pytest
+
+EMAIL = 'testuser@gmail.com'
+USERNAME = 'testuser'
+PASSWORD = 'testpassowrd'
+DOG_NAME = 'kliford'
+FIRST_NAME = 'NEW'
+LAST_NAME = 'USER'
+PHONE_NUMBER = 389450357
+DOG_RACE = 'lavrador'
+DOG_PICTURE_URL = 'http....'
+DOG_AGE = 10
+DOG_WEIGHT = 6
+DOG_GENDER = 'M'
+
+
+@pytest.fixture
+def dog_owner_0():
+    return User(email=EMAIL,
+                username=USERNAME,
+                password=PASSWORD,
+                dogowner=Dogowner(dog_name=DOG_NAME,
+                                  first_name=FIRST_NAME,
+                                  last_name=LAST_NAME,
+                                  phone_number=PHONE_NUMBER,
+                                  dog_race=DOG_RACE,
+                                  dog_picture_url=DOG_PICTURE_URL,
+                                  dog_age=DOG_AGE,
+                                  dog_weight=DOG_WEIGHT,
+                                  dog_gender=DOG_GENDER
+                                  )
+                )
+
+
+@pytest.mark.django_db
+class TestDogOwnerModel:
+    def test_create_dogowner(self, dog_owner_0):
```

Comment:
The tests lack coverage of edge cases for example:
what happens if you try to add the same user twice?
Think of them and add more tests accordingly.